lib/cog/repository/bundles.ex

```diff
@@ -50,6 +50,17 @@ defmodule Cog.Repository.Bundles do
     end
   end
 
+
+  # Deletes the embedded bundle when Mix.env == :dev so Cog can recreate the bundle
+  # at startup.
+  defp __reset_embedded() do
+    if Mix.env == :dev do
+      Logger.info("Dev mode detected. Resetting embedded bundle.")
+      Repo.delete_all(from b in Bundle,
+                   where: b.name == ^Cog.embedded_bundle)
```

Comment:
I think we might need to delete the current bundle version instead of the bundle itself... otherwise I suspect we lose all permissions, and then any users in the system will lose the ability to do anything